server/ot_rpc_test.go

```diff
@@ -50,6 +50,10 @@ func (st *stubTranscoder) Transcode(ctx context.Context, md *core.SegTranscoding
 	return testRemoteTranscoderResults, nil
 }
 
+func (st *stubTranscoder) EndTranscodingSession(sessionId string) {
+
+}
+
```

Comment:
Is it empty function?

Reply:
It's a stub for testing - nothing to tear down here.